presidio-analyzer/presidio_analyzer/pattern_recognizer.py

```diff
@@ -16,7 +16,7 @@
 class PatternRecognizer(LocalRecognizer):
 
     def __init__(self, supported_entity, name=None,
-                 supported_language='en', patterns=None,
+                 supported_language='all', patterns=None,
```

Comment:
We once had a discussion about this and decided that each recognizer is language specific, even international ones like credit card. Main reason is the context which is always language specific. So even if we added French which has a spaCy model we can use, we would still need to figure out which context words to use.

Reply:
but if we go this way we still get a strong set of multilanguage recognizers.
we can open this for discussion again or I can revert. both ok

Reply:
I think we should revert. If someone wants to add real support for a new language, they'll be in trouble with all the context words definition.

Reply:
ok.

Reply:
Done @omri374